src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -91,6 +98,8 @@ private void PopulateView()
 
                             owners = owners.Union(organizationLogins).ToArray();
                         });
+                    }).Finally(task => {
+                        isBusy = false;
```

Comment:
Hmm, the more I look at it, I realize this `Finally` is incorrect in other ways..
The `GetCurrentUser` callback has two different execution paths, one is synchronous the other is async.

https://github.com/github-for-unity/Unity/blob/0e489246486557981c314f7c0c82a65bc65fe1b3/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PublishView.cs#L73-L103